kroxylicious-operator-dist/pom.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright Kroxylicious Authors.
+
+    Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.kroxylicious</groupId>
+        <artifactId>kroxylicious-parent</artifactId>
+        <version>0.14.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>kroxylicious-operator-dist</artifactId>
+    <name>Kubernetes Operator distribution</name>
+    <description>Assembles the operator assets for release</description>
+    <packaging>pom</packaging>
```

Comment:
Using `pom` packaging is not ideal, but `jar` (the default) meant we generated an empty jar in `target`, which seemed funky.

Reply:
Its the least ugly option out of the defaults: `pom, jar, maven-plugin, ejb, war, ear, rar` How the hell RAR is there but not zip...